src/frame_compare/services/report/assets/viewer.css

```
@@ -2,25 +2,28 @@
+    --border-hover: rgba(255, 255, 255, 0.15);
+    --border-active: rgba(56, 189, 248, 0.4);
     --danger: #e55353;
     --success: #4caf50;
-    --divider: #5ba4e6;
+    --divider: #38bdf8;
 
     /* Spacing & Sizes */
-    --font-sans: "Inter", "SF Pro Text", "Segoe UI Variable Text", "Segoe UI", system-ui, -apple-system, "BlinkMacSystemFont", sans-serif;
-    --font-mono: "SF Mono", Monaco, "Cascadia Code", Consolas, monospace;
+    --font-sans: "Inter", "SF Pro Text", "Segoe UI Variable Text", system-ui, -apple-system, sans-serif;
+    --font-mono: "Cascadia Code", "SF Mono", "Segoe UI Mono", Consolas, monospace;
```

Stylelint (17.12.0)

[error] Expected "Consolas" to be "consolas" (value-keyword-case)

(value-keyword-case)